fuel-indexer/indexer/src/manifest.rs

```diff
@@ -5,6 +5,7 @@ pub struct Manifest {
     pub namespace: String,
     pub graphql_schema: String,
     pub wasm_module: String,
+    pub start_block: Option<u64>,
     pub handlers: Vec<Handler>,
     pub test_events: Vec<Event>,
 }
```

Comment:
Shouldn't we also subscribe these handlers to a particular contract id?

Reply:
Yeah, definitely want to do that. I was thinking I'd do it once we get event handling. Right now I'm sending in tests that have contract id set to 0.